integration-tests/tests/cargo/mod.rs

```diff
@@ -69,8 +69,8 @@ fn test_dependency_default_features() -> cargo_near::CliResult {
         pub fn foo(&self, a: bool, b: u32) {}
     };
 
-    assert_eq!(abi_root.body.functions.len(), 1);
-    let function = &abi_root.body.functions[0];
+    assert_eq!(abi_root.body.functions.len(), 2);
+    let function = &abi_root.body.functions[1];
```

Comment:
all of changes, similar to these 2 lines, are caused by abi.json generated change from
```json
{
  "schema_version": "0.4.0",
  "metadata": {
    "name": "test_dependency_default_features",
    "version": "0.1.0"
  },
  "body": {
    "functions": [
      {
        "name": "foo",
        "kind": "view",
        "params": {
          "serialization_type": "json",
          "args": [
            {
              "name": "a",
              "type_schema": {
                "type": "boolean"
              }
            },
            {
              "name": "b",
              "type_schema": {
                "type": "integer",
                "format": "uint32",
                "minimum": 0.0
              }
            }
          ]
        }
      }
    ],
    "root_schema": {
      "$schema": "http://json-schema.org/draft-07/schema#",
      "title": "String",
      "type": "string"
    }
  }
}
```
to 
```json
{
  "schema_version": "0.4.0",
  "metadata": {
    "name": "test_dependency_default_features",
    "version": "0.1.0"
  },
  "body": {
    "functions": [
      {
        "name": "contract_source_metadata",
        "kind": "view"
      },
      {
        "name": "foo",
        "kind": "view",
        "params": {
          "serialization_type": "json",
          "args": [
            {
              "name": "a",
              "type_schema": {
                "type": "boolean"
              }
            },
            {
              "name": "b",
              "type_schema": {
                "type": "integer",
                "format": "uint32",
                "minimum": 0.0
              }
            }
          ]
        }
      }
    ],
    "root_schema": {
      "$schema": "http://json-schema.org/draft-07/schema#",
      "title": "String",
      "type": "string"
    }
  }
}
```
the second index depends on the alphabetic order of `"add"` or `"foo"` with respect to `"contract_source_metadata"`